### ISSUE TITLE
Some tips

### ISSUE THREAD

**Issue by DmitrijVC** [Vazity1337/Discord-Nitro-Generator#19]
Check the codes by using this endpoint `https://discordapp.com/api/{version}/entitlements/gift-codes/{code}` and redeem them by using this one `https://discordapp.com/api/{version}/entitlements/gift-codes/{code}/redeem`. There are some tricks to bypass the "429" requests limit as for example using proxies or bot token `"authorization": f"Bot {token}"`.

I see no point in generating random codes into a file other than using them later by another program that could check them. I strongly doubt that You would like to change it, but [here](https://github.com/DmitrijVC/SomeShittyPythonScript) is my shitty code that does what I described above, You can base on it.

**Comment by gdfrjd2WasTaken** [comment]
can someone help me generate the code i dont know how to do it

**Comment by Vopiid** [comment]
please someone explain o me better on how to use these codes

**Comment by DmitrijVC** [comment]
> can someone help me generate the code i dont know how to do it

> please someone explain o me better on how to use these codes

Nitro code snipers/generators are useless. There is very low chance of you getting a valid code.

**Comment by Vazity1337** [comment]
Thanks for this suggestion, I might make this in c# soon

**Comment by Deenarz** [comment]
I don't understand how to generate the code, could someone message me one please or tell me how to get codes

**Comment by Deenarz** [comment]
> I don't understand how to generate the code, could someone message me one please or tell me how to get codes

I got the codes up but when I enter the link into my browser it sends me to discord website, loading, and saying "retrieving your gift from cyberspace", if let it sit for about 20 minutes and nothing has happened

**Comment by DmitrijVC** [comment]
> > I don't understand how to generate the code, could someone message me one please or tell me how to get codes
> 
> I got the codes up but when I enter the link into my browser it sends me to discord website, loading, and saying "retrieving your gift from cyberspace", i let it sit for about 20 minutes and nothing has happened

That's because these codes are generated randomly and aren't valid. The chance of you getting a working one is that low that You would more likely to win in a lottery.

**Comment by Imagin3Better** [comment]
Why do people think this would work. If you read the code, you would know that the program is generating random strings.

**Comment by goople-tech** [comment]
yeah

**Comment by HyperCuube** [comment]
I like how everyone assumes that there is a better way to passively get nitro. There are better active methods like scamming discord, but you can take a good gen and checker combo and host it on repl.it or something and then check it once a day for any codes. OR you hook that gen/checker combo up to a discord bot and get the codes in your DMs.

**Comment by DmitrijVC** [comment]
> I like how everyone assumes that there is a better way to passively get nitro. There are better active methods like scamming discord, but you can take a good gen and checker combo and host it on repl.it or something and then check it once a day for any codes. OR you hook that gen/checker combo up to a discord bot and get the codes in your DMs.

Good luck with it, maybe after some years, if you are lucky, you will get a valid code.
The best option is to just scam someone.

**Comment by TKperson** [comment]
I totally agree with @DmitrijVC. I have saw some of @HyperCuube other posts on other token/nitro generators. I have also replied to some of them, you should go check them. There's no way that you will get from this (maybe some decades later). Also, I have done some math to predict how big this nitro token is. See https://github.com/lnxcz/nitro-generator/issues/8

> "authorization": f"Bot {token}" can bypass the rate limiting

I'm not sure about bypassing the nitro part of rate limiting, but I'm sure of this method will never bypass anything in most of the discord (maybe all) http requests because rate limiting is based on ip. However, I do think can "bypass" by using a proxy. Well using a proxy is not really bypass but to reset your rate limiting as your ip get changed....

**Comment by znqi** [comment]
> Check the codes by using this endpoint `https://discordapp.com/api/{version}/entitlements/gift-codes/{code}` and redeem them by using this one `https://discordapp.com/api/{version}/entitlements/gift-codes/{code}/redeem`. There are some tricks to bypass the "429" requests limit as for example using proxies or bot token `"authorization": f"Bot {token}"`.
> 
> I see no point in generating random codes into a file other than using them later by another program that could check them. I strongly doubt that You would like to change it, but [here](https://github.com/DmitrijVC/SomeShittyPythonScript) is my shitty code that does what I described above, You can base on it.

I added that suggestion as well as proxy system. You can check my forked one. I also create a PR
